projects/storefrontapp/src/app/spartacus/spartacus-features.module.ts

```diff
@@ -114,6 +114,9 @@ if (environment.b2b) {
 if (environment.cdc) {
   featureModules.push(CdcFeatureModule);
 }
+// if (environment.cdp) {
```

Comment:
is this commented out code portion expected?

Reply:
it is not needed as of now, as CdpFeatureModule doesnot have any modules to import.